SwiftWisdom/Core/StandardLibrary/Numbers/BinaryFloatingPoint+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  BinaryFloatingPoint+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Stephen Wong on 5/1/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension BinaryFloatingPoint {
+    // https://codereview.stackexchange.com/questions/142748/swift-floatingpoint-rounded-to-places
+    public func ip_round(toDecimalPlaces places: Int) -> Self {
```

Comment:
Should it be `ip_rounded`? I think `ip_round` indicates it's a mutating function.

We could deprecate `ip_round` and forward it to an `ip_rounded` function.

Reply:
Or leave this one, make it `mutating` and add the new `ip_rounded`

Reply:
Which I guess we would need to deprecate anyway since its a different type of function

Reply:
Good point, might be nice to have the mutating version... would you mind adding that here too @MaxHasADHD? (I also don't mind turning that into a separate PR)

I don't know if swift has a way of deprecating and then repurposing a function name... I think the error that will be thrown from using a func with the wrong signature will be enough to indicate to the developer what to do. Other thoughts?

Reply:
Maybe mark the original version (with the now-bad signature) with some flavor of the `@available` attribute to show that it's no longer available?

Reply:
Based on https://docs.swift.org/swift-book/ReferenceManual/Attributes.html , maybe something like

`@available(*, unavailable, renamed: "ip_rounded(toDecimalPlaces places: Int) -> Self")`

That isn't a valid Obj-C-like method signature. Clean it up as necessary.